server/pbench/bin/pbench-backup-to-s3

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+
+import boto3
+import botocore
+import sys
+import os
+import configtools
+import glob
+import base64
+import sh
+
+options = []
+opts, args = configtools.parse_args(options)
+conf, files = configtools.init(opts)
+
+# configuration file
+endpoint_url = conf.get('pbench-server-backup', 'endpoint_url')
+access_key_id = conf.get('pbench-server-backup', 'access_key_id')
+secret_access_key = conf.get('pbench-server-backup', 'secret_access_key')
+bucket_name = conf.get('pbench-server-backup', 'bucket_name')
+archive = conf.get('pbench-server', 'pbench-archive-dir')
+backup = conf.get('pbench-server', 'pbench-backup-dir')
+
+# make connection to s3
+# s3 = boto3.resource('s3', endpoint_url = '%s' % endpoint_url)
+
+s3 = boto3.resource('s3', aws_access_key_id='%s' % access_key_id,
+                    aws_secret_access_key='%s' % secret_access_key, endpoint_url='%s' % endpoint_url)
+
+
+def rsync_archive_backup(archive, backup):
+
+    if not os.path.isdir(archive):
+        raise ValueError('Archive %s not found.' % archive)
+
+    if not os.path.isdir(backup):
+        raise ValueError('Backup %s not found.' % backup)
+
+    exclude = ['--exclude=_QUARANTINED',
```

Comment:
Could you consider fetching the list of state directories to exclude from the environment variable, https://github.com/distributed-system-analysis/pbench/blob/master/server/pbench/bin/pbench-base.sh#L137?

Something like:

```
    exclude = os.environ['EXCLUDE_DIRS'].split()
```

And then modifying `pbench-base.sh` to `export EXCLUDE_DIRS`?